### DISCUSSION TITLE
🥇  What is your favorite Sushi ? （好きな寿司は何ですか？）🥇

### DISCUSSION THREAD

**Discussion by YashyHoby** [NAIST-SE/AttractingContributors#1177] [2024 Attraction Contributions]
Tell me your favorite Sushi, please! ↓↓↓
https://github.com/YashyHoby/AttractingContributors_Sushi

**Comment by ShunHattori** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.
https://github.com/ShunHattori/AttractingContributors_EssentialGadget.git
Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1148

**Comment by honamikoyama** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.
https://github.com/honamikoyama/Travel.git
Discussion link:  https://github.com/NAIST-SE/AttractingContributors/discussions/1182

**Comment by Idescrew** [comment]
I just made a PR on your repo.
Could you contribute to my repo? Thanks in advance.

https://github.com/Idescrew/AttractingContributors

**Comment by marshr00m** [comment]
I submitted a pull request. I would appreciate any additions to my list!
https://github.com/marshr00m/AttractingContributors-Where-would-you-like-to-travel
